### PR TITLE
Add facebook_metrics.csv to analysis

### PR DESCRIPTION
**Pull request by as2388**
As discussed offline, this is complex because we're essentially preparing this piece of the automated analysis in the fetch_raw_data stage. This PR should hopefully read as a straightforward function that's pulling this data, plus the plumbing needed to copy this through to automated analysis.

If this looks good I'd like to start moving the reusable facebook components into a new SocialMediaTools repository.

### CODE REVIEW

File: automated_analysis.py [hunk 1]
```diff
@@ -36,6 +38,9 @@
                         help="Path to a JSONL file to read the TracedData of the messages data from")
     parser.add_argument("individuals_json_input_path", metavar="individuals-json-input-path",
                         help="Path to a JSONL file to read the TracedData of the messages data from")
+    parser.add_argument("engagement_metrics_input_dir", metavar="engagement-metrics-input-dir",
```
Comment (IsaackMwenda) [inline automated_analysis.py:41]: will this contain other non-social media engagement files?

File: docker-run-automated-analysis.sh [hunk 2]
```diff
@@ -25,11 +25,11 @@ done
 
 
 # Check that the correct number of arguments were provided.
-if [[ $# -ne 5 ]]; then
+if [[ $# -ne 6 ]]; then
     echo "Usage: ./docker-run-automated-analysis.sh
     [--profile-cpu <profile-output-path>]
     <user> <pipeline-configuration-file-path> <messages-traced-data>
-    <individuals-traced-data> <automated-analysis-output-dir>"
+    <individuals-traced-data> <metrics-dir> <automated-analysis-output-dir>"
```
Comment (IsaackMwenda) [inline docker-run-automated-analysis.sh]:
```suggestion
    <individuals-traced-data> <engagement-metrics-dir> <automated-analysis-output-dir>"
```

**Review by IsaackMwenda — APPROVED** [review]

File: automated_analysis.py (hunk 1 above)
Comment (as2388) [reply automated_analysis.py:41]: Good question - I'm not aware of anything else immediately upcoming, but I stuck with a generic directory that we could add other files to in future in order to make it easier to do so should it become necessary, and to make it easier to handle the sms case which currently has no such export.

File: docker-run-automated-analysis.sh (hunk 2 above)
Comment (as2388) [reply docker-run-automated-analysis.sh]: thanks

**Pull request merged**